Compilers.sln

```diff
@@ -367,22 +367,22 @@ Global
 		{7FE6B002-89D8-4298-9B1B-0B5C247DD1FD}.Release|x86.Build.0 = Release|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|ARM.ActiveCfg = Debug|ARM
-		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|ARM.Build.0 = Debug|ARM
+		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|ARM.ActiveCfg = Debug|Any CPU
+		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|ARM.Build.0 = Debug|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
-		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|x64.ActiveCfg = Debug|x64
-		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|x64.Build.0 = Debug|x64
+		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|x64.Build.0 = Debug|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Debug|x86.Build.0 = Debug|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|Any CPU.Build.0 = Release|Any CPU
-		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|ARM.ActiveCfg = Release|ARM
-		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|ARM.Build.0 = Release|ARM
+		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|ARM.ActiveCfg = Release|Any CPU
+		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|ARM.Build.0 = Release|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|Mixed Platforms.Build.0 = Release|Any CPU
-		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|x64.ActiveCfg = Release|x64
-		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|x64.Build.0 = Release|x64
+		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|x64.ActiveCfg = Release|Any CPU
+		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|x64.Build.0 = Release|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|x86.ActiveCfg = Release|Any CPU
 		{F7712928-1175-47B3-8819-EE086753DEE2}.Release|x86.Build.0 = Release|Any CPU
 		{4371944A-D3BA-4B5B-8285-82E5FFC6D1F9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Are changes in this file related to the fix? If not, consider reverting and submitting as a separate PR.